### DISCUSSION TITLE
cannot use space as leader in vim mode

### DISCUSSION THREAD

**Discussion by nagy-nabil** [zed-industries/zed#16474] [Feature Requests]
based on this issue https://github.com/zed-industries/zed/issues/7270, the problem in front of me should be fixed, so I didn't want to open an issue before making sure my configuration was correct

i want to use `space d s` to open the current buffer symbols menu, so i have the following configuration, but every time i click on space it doesn't wait for my next input instead it moves the cursor one step ahead, so i cannot use any shortcut that starts with space
here's my config
```json
	{
		"context": "Editor && VimControl && !VimWaiting && !menu",
		"bindings": {
			"space d s": "outline::Toggle"
		}
	}
```

zed version `Zed 0.148.1`
os `Ubuntu 24.04 LTS`

if I should provide any logs or anything please let me know

**Comment by universalmind303** [comment]
it should work if you use this context. `"context": "Editor && (vim_mode == normal || vim_mode == visual) && !VimWaiting && !menu",`

I'm using space leader with this configuration just fine. 

```json
  {
    "context": "Editor && (vim_mode == normal || vim_mode == visual) && !VimWaiting && !menu",
    "bindings": {
      "space": null,
      "space o": "tab_switcher::Toggle",
      "space space": "command_palette::Toggle",
      "space f": "file_finder::Toggle",
      "space t": "workspace::NewCenterTerminal",
      "space /": "workspace::NewSearch",
      "space shift-d": "diagnostics::Deploy",
      "space d d": "diagnostics::Deploy",
      "space d w": "diagnostics::ToggleWarnings",
      "space d o": "editor::GoToDiagnostic",
      "space d n": "editor::GoToPrevDiagnostic",
      "space s": "outline::Toggle",
      "space shift-s": "project_symbols::Toggle",
      }
    }
```

**Reply by nagy-nabil** [reply]
okay i will test it thank you

**Reply by nagy-nabil** [reply]
okay it works thank you
